finch/processes/__init__.py

```diff
@@ -25,22 +24,36 @@
 
 
 def get_indicators(module):
-    """For a given module, return the children that are instances of xclim.utils.Indicator."""
-    return [o for o in module.__dict__.values() if isinstance(o, Indicator)]
+    """For all modules or classes listed, return the children that are instances of registered Indicator classes.
+
+    module : A xclim module.
+    """
+    from collections import OrderedDict
+    from xclim.core.indicator import registry
+
+    out = {}
+    for key, val in module.__dict__.items():
+        if hasattr(val, "identifier"):
+            name = val.identifier.upper()
+            if name in registry:
+                out[key] = val
+
+    return OrderedDict(sorted(out.items())).values()
```

Comment:
Is there a reason we are restricting the parsing to a module? With 0.21, we have  `.get_instance()` and  `.realm`. Doing:
```
indicators = [ind.get_instance() for name,  ind in registry.items() if ind.realm == 'atmos' and name not in not_implemented]
```
would simplify the process, and would open the door for externally-declared indicators, which was what the registry was implemented for!

Reply:
Indeed. Fixed.